### PR TITLE
Fix dynamo benchmark runner for torchbench skip sets

### PR DESCRIPTION
**Pull request by chuanqi129**
Fix dynamo benchmark runner for torchbench skip sets, which introduced by PR #118032 

This runner.py script is still used in the [Inductor CPU Performance Dashboard](https://github.com/pytorch/pytorch/issues/93531) regular test

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @chenyang78 @aakhundov @kadeng

### CODE REVIEW

**Review by jgong5 — APPROVED** [review]

**Comment by chuanqi129** [comment]
Hi @ysiraichi @ezyang , could you please help to review this PR when you have free time? It will impact the inductor cpu dashboard passrate of torchbench. Thanks very much~

**Comment by ezyang** [comment]
@ysiraichi could you please review this. I'll merge it after you approve

**Review by ysiraichi — APPROVED** [review]
LGTM. @ezyang done.

**Review by ezyang — APPROVED** [review]

**Pull request closed without merging**
